plugin/forward/setup.go

```diff
@@ -157,7 +157,7 @@ func parseStanza(c *caddy.Controller) (*Forward, error) {
 		if f.opts.forceTCP && transports[i] != transport.TLS {
 			f.proxies[i].health.SetTCPTransport()
 		}
-		f.proxies[i].health.SetDomain(f.opts.hcDomain)
+		f.proxies[i].health.SetDomain(plugin.Name(f.opts.hcDomain).Normalize())
```

Comment:
Need to make sure that result of normalize is not empty string. 
 probably better to do it when getting the value from config, in parseBlock.